README.md

```diff
@@ -57,7 +57,7 @@ TensorFlow model scripts and running them the usual way:
 
         import ngraph_bridge
 
-Examples on how to use ngraph_bridge is here : https://github.com/NervanaSystems/ngraph-tf/tree/master/examples
+Detailed examples on how to use ngraph_bridge is located in the [examples] directory.
```

Comment:
`git checkout v0.12.0-rc6` needs to change to `git checkout v0.12.0`

Reply:
Done.